Cargo.toml

```diff
@@ -25,7 +25,7 @@ rpath = true
 rpath = true
 
 [workspace.dependencies]
-alloy-eips = { version = "0.11.1", default-features = false }
+alloy-eips = { version = "0.13.0", default-features = false }
```

Comment:
This is a bit risky as Foundry is still on `alloy` 0.12, but I assume it's necessary because of REVM 21. So having `alloy-eips = 0.12` wouldn't work, right?

Reply:
No, it's unfortunately required 😔

Reply:
ok thanks for checking!